src/sql/src/plan/statement.rs

```diff
@@ -665,6 +665,18 @@ fn kafka_sink_builder(
         bail!("replication factor for sink topics has to be greater than zero");
     }
 
+    let consistency_value_schema = match with_options.remove("consistency") {
```

Comment:
so in the future we may have multiple consistency formats .... maybe we want the user to name it here. counterpoint - what would be a good name? "debezium" jumps to mind but then we have "debezium" the envelope and "debezium" the consistency format and that's confusing

Reply:
Right now, (and I suspect this might be the best way going forward, unless we create a separate CREATE CONSISTENCY SOURCE command) to "pair" the consistency topic with the FORMAT/ENCODING specification. Right now, in the timestamper.rs, if we see a Debezium envelope, we assume that the consistency topic is going to follow Debezium + Avro, if we see a regular avro encoding, w'ere going to assume that the consistency format is going to be the basic one (with Avro formatting, etc.)